kolibri/core/apps.py

```diff
@@ -161,6 +165,12 @@ def check_redis_settings():
                     "Problematic Redis settings detected, please see Redis configuration "
                     "documentation for details: https://redis.io/topics/config"
                 )
+
+        except ConnectionError as e:
+            logger.warning("Unable to connect to Redis: {}".format(str(e)))
+
+            raise RedisConnectionError("Unable to connect to Redis: {}".format(str(e)))
```

Comment:
The only slight tweak I might suggest would be to use the six `raise_from` utility here, example from recent work by @bjester https://github.com/learningequality/kolibri/blob/d43ab6dfec79512dad1e4858c68024e1adee3fc3/kolibri/core/tasks/job.py#L163

Reply:
Let me add this